lib/diff.ml

```diff
@@ -354,19 +380,22 @@ let class_type_item ~typing_env ~name
                      { reference = ref_class_type; current = curr_class_type };
                }))
 
-let rec items ~reference ~current =
-  let env = Typing_env.for_diff ~reference ~current in
-  let ref_items = extract_items reference in
-  let curr_items = extract_items current in
+let rec items ~reference ~current ~typing_env =
+  let ref_items =
+    extract_items reference ~sub:false typing_env.Typing_env.subst
+  in
+  let curr_items =
+    extract_items current ~sub:true typing_env.Typing_env.subst
```

Comment:
It'd be great to have a comment here explaining why this one needs to have `~sub:true`. It would also probably be great to have a doc comment on `extract_items` explaining what the function does and what the `~sub` argument controls.

Reply:
I updated the comment in the `lib/typing_env.mli`, I think that's enough to understand `sub` flag in `extract_items`.